go.mod

```diff
@@ -286,3 +284,6 @@ replace github.com/Masterminds/goutils => github.com/Masterminds/goutils v1.1.1
 
 // https://bugzilla.redhat.com/show_bug.cgi?id=2045880
 replace github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.12.1
+
+// after the openshift/api PR #1390 is merged, this needs to remove
+replace github.com/openshift/api => github.com/nutanix-cloud-native/openshift-api v0.0.0-20230207165630-17ffc12900d6
```

Comment:
are we intending to point to nutanix-cloud-native here?

Reply:
This is temporary, since the openshift/api PR https://github.com/openshift/api/pull/1390 has not been merged yet (waiting for the QE label). Once the api PR is merged, I will change this accordingly.

Reply:
Addressed.